.github/workflows/build-test.yml

```diff
@@ -44,10 +44,15 @@ jobs:
         runs-on: ubuntu-latest
         strategy:
             matrix:
-                image: [ 'nodejs-centos6-default', 'nodejs-centos7-node12', 'nodejs-centos7-fips' ]
                 cloud: [ 'AWS', 'AZURE', 'GCP' ]
```

Comment:
why we change this?

Reply:
The v3 AWS SDK requires a node version >= 12. These dockers use centos 6 which is not compatible with later versions of node so it results in test failures. 

When running the tests on centos 6, this is the warning stating the node version is unsupported by the package: https://github.com/snowflakedb/snowflake-connector-nodejs/actions/runs/2884223384/jobs/4583575600#step:4:232

And this is the resulting error from the test run: https://github.com/snowflakedb/snowflake-connector-nodejs/actions/runs/2884223384/jobs/4583575600#step:4:403

I tried to update to the latest node version but centos6 is incompatible. So instead of using the centos images, the latest ubuntu build provided by github is now used instead for testing.